src/couch_replicator_httpc.erl

```diff
@@ -162,7 +167,11 @@ process_stream_response(ReqId, Worker, HttpDb, Params, Callback) ->
     receive
     {ibrowse_async_headers, ReqId, Code, Headers} ->
         case list_to_integer(Code) of
+        429 ->
+            Timeout = couch_replicator_ratelimiter:limit(),
```

Comment:
I ended up with `max_interval` as the rest of the code and description uses interval